src/app/package-json.js

```diff
@@ -132,15 +132,21 @@ const packageJSON = (current, context) => {
         'build:js:uglify'
       ]),
 
+      // Babel is a run in a distinct step (vs. using babelify) because we want
+      // the transpiled code to be what's provided to module consumers using
+      // Node or Browserify.
       'build:js:babel': 'babel src -d es5',
 
       'build:js:bannerize': scriptify([
         'bannerize dist/%s.js',
         '--banner=scripts/banner.ejs'
       ]),
 
+      // The browserify-shim transform is included ONLY in the build step as
+      // we do not want consuming projects to receive a shimmed module - if we
+      // shim always, we cause obscure errors!
       'build:js:browserify': scriptify([
-        'browserify . -s %s -o dist/%s.js'
+        'browserify . -t browserify-shim -s %s -o dist/%s.js'
```

Comment:
this needs to be changes to a global transform: `-g browserify-shim` rather than `-t browserify-shim`